### PR TITLE
C++ front-end Bert-loss function: Reshape inputs for SoftmaxCrossEntropyLoss instead of transposing them to improve performance.

### PR DESCRIPTION
**Pull request by codemzs**
This change improves the performance of BERT-LOSS function for C++ front-end by reshaping inputs for SoftmaxCrossEntropyLoss instead of transposing them.

### CODE REVIEW

**Comment by SherlockNoMad** [comment]
Thanks for the quick fix. 
For future PRs, please add the labels correspondingly. I was using the label filter to find training PRs.
